.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    env:
+      CC: gcc-8
+      CXX: g++-8
+    steps:
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.8'
+    - name: Install dependencies
+      run: |
+        sudo apt-get install -y ninja-build nasm gcc-8 g++-8
```

Comment:
```suggestion
        DEBIAN_FRONTEND=noninteractive sudo apt-get update
        DEBIAN_FRONTEND=noninteractive sudo apt-get install -y ninja-build nasm gcc-8 g++-8
```

Reply:
Thansk for the review! What does DEBIAN_FRONTEND=noninteractive exactly do?

Reply:
https://askubuntu.com/questions/556385/how-can-i-install-apt-packages-non-interactively
```none
noninteractive
      This  is  the anti-frontend. It never interacts with you at all,
      and makes the default answers be  used  for  all  questions.  It
      might  mail  error messages to root, but that's it; otherwise it
      is completely silent and unobtrusive,  a  perfect  frontend  for
      automatic installs. If you are using this front-end, and require
      non-default answers to questions, you will need to  preseed  the
      debconf  database;  see  the section below on Unattended Package
      Installation for more details.
```

Makes it so that it will never ask you anything